client/me/help/help-contact/index.jsx

```diff
@@ -26,13 +26,13 @@ import notices from 'notices';
 import analytics from 'lib/analytics';
 import { isOlarkTimedOut } from 'state/ui/olark/selectors';
 import { isCurrentUserEmailVerified } from 'state/current-user/selectors';
-import { isHappychatAvailable } from 'state/happychat/selectors';
+import { isHappychatAvailable } from 'extensions/happychat/state/selectors';
```

Comment:
Same here, if we aren't able to remove the direct references to the extension, we should at least comment prominently that it needs to be resolved.